docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -90,44 +100,45 @@ For example, the application processes only 4 video streams, so there is no need
+  - Balance the `timeout` value vs. the batch size. For example, in many cases, having a smaller `timeout` value/batch size may yield better performance than having a larger batch size with a `timeout` value that is not large enough to accommodate the full number of the required requests.
+  - When Automatic Batching is enabled, the `timeout` property of `ov::CompiledModel` can be changed anytime, even after the loading/compilation of the model. For example, setting the value to 0 disables Auto-batching effectively, as the collection of requests would be omitted.
+  - Carefully apply Auto-batching to the pipelines. For example, in the conventional "video-sources -> detection -> classification" flow, it is most beneficial to do Auto-batching over the inputs to the detection stage. The resulting number of detections is usually fluent, which makes Auto-batching less applicable for the classification stage.
 
 The following are limitations of the current implementations:
- - Although less critical for the throughput-oriented scenarios, the load-time with auto-batching increases by almost 2x.
- - Certain networks are not safely reshape-able by the "batching" dimension (specified as 'N' in the layouts terms). Also, if the batching dimension is not zero-th, the auto-batching is not triggered _implicitly_ by the throughput hint.
- -  The _explicit_ notion, for example, "BATCH:GPU", uses the relaxed dimensions tracking, often making the auto-batching possible. For example, this trick unlocks most **detection networks**.
- - - When *forcing* the auto-batching via the explicit device notion, make sure to validate the results for correctness.   
- - Performance improvements happen at the cost of the memory footprint growth, yet the auto-batching queries the available memory (especially for the dGPUs) and limits the selected batch size accordingly.
+- Although it is less critical for the throughput-oriented scenarios, the load time with Auto-batching increases by almost double.
+ - Certain networks are not safely reshapable by the "batching" dimension (specified as `N` in the layout terms). Besides, if the batching dimension is not zeroth, Auto-batching will not be triggered "implicitly" by the throughput hint.
+ -  The "explicit" notion, for example, `BATCH:GPU`, using the relaxed dimensions tracking, often makes Auto-batching possible. For example, this method unlocks most **detection networks**.
+ - When *forcing* Auto-batching via the "explicit" device notion, make sure that you validate the results for correctness.   
+ - Performance improvements happen at the cost of the growth of memory footprint. However, Auto-batching queries the available memory (especially for dGPU) and limits the selected batch size accordingly.
 
  
-### Configuring the Automatic Batching
-Following the OpenVINO convention for devices names, the *batching* device is named *BATCH*. The configuration options are as follows:
+## Configuring Automatic Batching
```

Comment:
As I commented earlier, we can move this section up to be under the enabling/disabling section.